packages/spor-theme-react/src/components/alert.ts

```diff
@@ -0,0 +1,116 @@
+import { border, createMultiStyleConfigHelpers } from "@chakra-ui/react";
+import { anatomy } from "@chakra-ui/theme-tools";
+import { focusVisible } from "../utils/focus-utils";
+
+const parts = anatomy("alert").parts(
+  "container",
+  "iconContainer",
+  "textContainer",
+  "title"
+);
+
+const helpers = createMultiStyleConfigHelpers(parts.keys);
+
+const config = helpers.defineMultiStyleConfig({
+  baseStyle: (props) => ({
+    container: {
+      display: "inline-flex",
+      alignItems: "center",
+      width: "fit-content",
+      justifyContent: ["space-between", "center"],
+      transitionDuration: "fast",
+      transitionProperty: "common",
+      fontSize: ["mobile.sm", "desktop.sm"],
+      border: 0,
+      whiteSpace: "nowrap",
+      px: 1,
+      borderRadius: "sm",
+      paddingLeft: 2,
+      paddingRight: 2,
+      paddingTop: 2,
+      paddingBottom: 2,
+      padding: 0.5,
+      position: "relative",
+      ...focusVisible({
+        focus: {
+          color: "white",
+          border: "2px solid",
+          borderColor: "darkGrey",
+        },
+        notFocus: { boxShadow: "none" },
+      }),
+    },
+    iconContainer: {
+      padding: 1,
+      color: "darkGrey",
+    },
+    textContainer: {
+      color: "darkGrey",
+      whiteSpace: "nowrap",
+    },
+    title: {
+      display: "-webkit-box",
+      WebkitBoxOrient: "vertical",
+      textAlign: "right",
+    },
+  }),
+  variants: {
+    warning: {
+      container: {
+        backgroundColor: "blonde",
+      },
+      _hover: {
+        backgroundColor: "primerose",
+      },
+      _active: {
+        backgroundColor: "cornsilk",
+      },
```

Comment:
Disse to skal sikkert ligge inni container-blokken?

Reply:
Nei, da kunne man ikke ser forskjellige på farge.